templates/preview.html

```diff
@@ -0,0 +1,141 @@
+{% extends 'layouts/_base.html' %}
+
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/accordion/_macro.njk" import onsAccordion %}
+
+{% set save_on_signout = true %}
+
+{% set breadcrumbs = {
+  "ariaLabel": 'Back',
+  "itemsList": [
+    {
+      "url": url_for("questionnaire.get_questionnaire"),
+      "id": "top-previous",
+      "text": _("Back"),
+      "attributes": {
+        "data-ga": 'click',
+        "data-ga-category": 'Navigation',
+        "data-ga-action": 'Previous link click'
+      }
+    }
+  ]
+}%}
+
+
+{% block main %}
+
+<h2 class="ons-u-mt-m">{{ _("Preview of the questions in this survey") }}</h2>
+
```

Comment:
```suggestion
<h1 class="ons-u-mt-m">{{ _("Preview of the questions in this survey") }}</h1>
```
This needs to be a h1, as theres no other h1 on this page